bff/hosts/Blazor/PerComponent/Hosts.Bff.Blazor.PerComponent/Program.cs

```diff
@@ -42,6 +42,10 @@
     {
         options.Cookie.Name = "__Host-blazor";
         options.Cookie.SameSite = SameSiteMode.Strict;
+        options.Events.OnValidatePrincipal += context =>
```

Comment:
Maybe this empty event isn't needed at all?

Reply:
correct.